### PR TITLE
GitHub Connector documentation update

### PR DESCRIPTION
**Pull request by colunira**
Our GitHub Connector documentation needed an update. Documents were scattered around repository, now they are in one place and linked in chart and github-connector directories. Also, some instructions were outdated or misleading (Connector's configuration).

### CODE REVIEW

File: chart/githubconnector/README.md [hunk 1]
```diff
@@ -1,34 +1,7 @@
...
-- [Overview](#overview)
-- [Prerequisites](#prerequisites)
-- [Installation in Kyma with Helm](#installing-in-kyma-using-helm)
-	- [Prerequisites](#prerequisites-1)
-	- [Steps](#steps)
+# Github Connector Chart<!-- omit in toc -->
 
 ## Overview
 The chart creates a GitHub Connector Deployment and a Namespace in Kyma.
 Moreover it creates a service, binds it to the newly created Namespace and exposes its API. Apart from that it creates an application.
```
Comment (majakurcius) [inline chart/githubconnector/README.md]:
```suggestion
Additionally, it creates a service, binds it to the newly created Namespace and exposes its API. Apart from that, it creates an application.
```
Add a comma after `Moreover`, `Additionally`, `Furthermore`, etc.

File: docs/github-connector/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# GitHub Connector
+
+## Overview
+
+The GitHub Connector is an additional tool for handling GitHub events in Kyma. It registers GitHub API and events in Kyma Application Registry. Then, it converts events incoming from connected GitHub webhooks into format acceptable by Kyma Event Bus and forwards them. For now it handles those events:
```
Comment (majakurcius) [inline docs/github-connector/README.md]:
```suggestion
The GitHub Connector is an additional tool for handling GitHub events in Kyma. It registers GitHub API and events in Kyma Application Registry. Then, it converts events incoming from connected GitHub webhooks to format acceptable by Kyma Event Bus and forwards them. For now it handles these events:
```

- `The GitHub Connector is an additional tool` - Additional to what? (Also, do we use `the` in front of GitHub connector? Just checking to make sure.)
- When you convert A **into** B, A becomes B. See [this](https://www.usingenglish.com/forum/threads/99446-convert-into-convert-to). 
- `to the format acceptable`, or `to a format acceptable`, if there are more acceptable formats
- @`For now`: As per [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/04-formatting.md):
  > **NOTE:** Do not use words such as "currently" or "now" to indicate that something is in the transitional phase of development. Avoid promising anything and mention only those components and functionalities that are already in use.

  You can rephrase it as: `The GitHub Connector handles:` or `The GitHub connector handles these events:`.

- `and forwards them` where?

File: docs/github-connector/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+- new pull request,
+- new issue,
+- new review on pull request.
```
Comment (majakurcius) [inline docs/github-connector/README.md]:
```suggestion
- new pull request review.
```

As per [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/04-formatting.md):
> Avoid ending bullet points with semicolons or commas.

Please modify all the bullet points.

File: docs/github-connector/configuration.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Configuring GitHub Connector
+
+## Overview
+
+This document describes how to correctly connect GitHub repository to the GitHub Connector installed in Kyma environment. After completion you will be able to handle events incoming from GitHub in lambdas.
```
Comment (majakurcius) [inline docs/github-connector/configuration.md]: As per guidelines, don't use future tense. See [here](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#tenses).

File: docs/github-connector/configuration.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+## Prerequisites
+
+- Kyma with the GitHub Connector [installed](/docs/github-connector/installation.md)
+- Connection to Kyma
+
+## Installation
+
+1. Find the newly created github-connector application and [bind it to the namespace](https://kyma-project.io/docs/components/application-connector/#tutorials-bind-an-application-to-a-namespace) of your choice.
```
Comment (majakurcius) [inline docs/github-connector/configuration.md]: Is `github-connector` the name of the app (file)? Or should it be `The GitHub Connector` application? Change if necessary.

File: docs/github-connector/configuration.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+2. Open settings of GitHub repository you want to connect to, go to `Webhooks` page and click `Add webhook`.
```
Comment (majakurcius) [inline docs/github-connector/configuration.md]:
```suggestion
2. Open the settings of the GitHub repository you want to connect to, go to `Webhooks` page and click `Add webhook`.
```

File: docs/github-connector/configuration.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+3. On the configuration page, fill the field `Payload URL` with exposed service URL (you can find it in `Kyma Console > {NAMESPACE} > APIs`) and add `/webhook` at the end of it.
```
Comment (majakurcius) [inline docs/github-connector/configuration.md]:
```suggestion
3. On the configuration page, fill the field `Payload URL` with exposed service URL (you can find it in `Kyma Console > {NAMESPACE} > APIs`) and add `/webhook` at the end of the URL.
```

File: docs/github-connector/configuration.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+4. Set other fields as follows:
+
+    - **Content type**: `application/json`
+    - **Secret**: `my-secret-key`
+    - **SSL verification**: `Disabled`
+
+    >**NOTE:** Secret is temporarily defined statically in code and SSL verification is disabled. It will be changed in the future.
```
Comment (majakurcius) [inline docs/github-connector/configuration.md]: As per guidelines:
> **NOTE:** Do not use words such as "currently" or "now" to indicate that something is in the transitional phase of development. Avoid promising anything and mention only those components and functionalities that are already in use.

See [this](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#terminology).

File: docs/github-connector/configuration.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+5. Select which events you would like to receive in GitHub Connector.
```
Comment (majakurcius) [inline docs/github-connector/configuration.md]:
```suggestion
5. Select which events you want to receive in GitHub Connector.
```

Is it `GitHub Connector` or `The GitHub Connector`? You use both in different files. Please check, adjust and be consistent.

File: docs/github-connector/configuration.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+6. Click `Add webhook`. You will be redirected back to webhooks page. There will be new webhook on list. Successful configuration results with green tick next to it.
```
Comment (majakurcius) [inline docs/github-connector/configuration.md]:
```suggestion
6. Click `Add webhook`. This redirects you back to the webhooks page. You can see a new webhook in the list. A successful configuration results in a green tick next to the new webhook.
```

File: docs/github-connector/examples/demoscenario.md [hunk 11]
```diff
@@ -10,8 +10,8 @@ This purpose of this guide is to show example usage of GitHub Connector, which a
 
 ## Prerequisites
 
-- Kyma with the GitHub Connector [installed](/chart/githubconnector/README.md).
-- GitHub webhook [configured](https://developer.github.com/webhooks/creating/) to deliver payload to the Connector's `/webhook` endpoint.
+- Kyma with the GitHub Connector [installed](/docs/github-connector/installation.md).
+- GitHub webhook [configured](/docs/github-connector/configuration.md) to deliver payload to the Connector's `/webhook` endpoint.
```
Comment (majakurcius) [inline docs/github-connector/examples/demoscenario.md]: Maybe specify which Connector you mean. (GitHub Connector?)

File: docs/github-connector/installation.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+## Installation in Kyma with Helm
+
+### Prerequisites
+
+- Connection to Kyma cluster
+- GitHub Connector Docker image
+
+### Steps
+
+1. Go to [Kyma repository](https://github.com/kyma-project/kyma) and run script `/installation/scripts/tiller-tls.sh` to get certificates needed for using Helm commands. By default they are stored in `~/.helm` directory. After that add `--tls` flag to every Helm command to authorize and authenticate yourself
```
Comment (majakurcius) [inline docs/github-connector/installation.md]:
```suggestion
1. Go to [Kyma repository](https://github.com/kyma-project/kyma) and run script `/installation/scripts/tiller-tls.sh` to get certificates needed for using Helm commands. By default they are stored in `~/.helm` directory. After that add the `--tls` flag to every Helm command to authorize and authenticate yourself
```

I don't like the `yourself` part. It sounds too colloquial and too general. What do you mean by that? To authenticate a user, an app? Please, specify.

File: docs/github-connector/installation.md [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+2. Go to `chart/githubconnector` directory. Run the command to install GitHub Connector:
```
Comment (majakurcius) [inline docs/github-connector/installation.md]:
```suggestion
2. Go to the `chart/githubconnector` directory. Run this command to install GitHub Connector:
```

File: docs/github-connector/installation.md [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+
+    ``` shell
+    helm install --set container.image={DOCKER_IMAGE} --set kymaAddress={KYMA_ADDRESS} -n {RELEASE_NAME} . --tls
+    ```
+
+    >**CAUTION:** Kyma address should be in the right format. It must consist of domain name, without dot  character at the beggining, for example `35.187.32.214.xip.io`
```
Comment (majakurcius) [inline docs/github-connector/installation.md]:
```suggestion
    >**CAUTION:** Make sure that the Kyma address is in the correct format. It must consist of the domain name, without the dot character at the beginning. For example, `35.187.32.214.xip.io`.
```
`It must` or `it consists`?
- Don't use `should`. See [this](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#terminology).
- Don't use `for example` in the middle of the sentence. See [this](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#terminology).
- I'd rephrase that:
  > It must consist of the domain name, **without the dot character at the beginning**.

  to something like:
  >  It consists of the domain name and omits the dot at the beginning. 

  if the dot is there in other cases and just here needs to be omitted.

File: docs/github-connector/installation.md [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+
+    >**NOTE:** To define Namespace in which chart should be installed add flag `--namespace`.
```
Comment (majakurcius) [inline docs/github-connector/installation.md]:
```suggestion
    >**NOTE:** To define the Namespace in which to install chart, add the flag `--namespace`.
```
Do we use `the` in front of `chart`? Do we capitalize it? (Just checking to make sure).

File: github-connector/README.md [hunk 16]
```diff
@@ -1,69 +1,5 @@
...
-* PullRequestReview Event
-* Star Event
-* Watch Event
+## Overview
 
-For the rest of the events, it will log out a message like this
-> ```
-> 2019/07/29 08:32:04 unknown event type: "fork"
-> ```
+The GitHub Connector is a tool for handling GitHub events. It is installed in Kyma environment and registers GitHub API and events in Kyma Application Registry. Then, it converts incoming events from connected GitHub webhooks into format acceptable by Kyma Event Bus and forwards them. See the [GitHub Connector documentation](/docs/github-connector) for more information.
```
Comment (majakurcius) [inline github-connector/README.md]:
```suggestion
The GitHub Connector is a tool for handling GitHub events. It is installed in the Kyma environment and it registers GitHub API and events in Kyma Application Registry. Then, it converts incoming events from connected GitHub webhooks into format acceptable by Kyma Event Bus and forwards them. See the [GitHub Connector documentation](/docs/github-connector) for more information.
```

See the second comment in this review and change 
> Then, it converts incoming events from connected GitHub webhooks into format acceptable by Kyma Event Bus and forwards them. 

to match the other file. 

Also, do we use `the` with `Kyma Application Registry` or do we treat it as a proper name?

File: docs/github-connector/README.md (hunk 2 above)
Comment (colunira) [reply docs/github-connector/README.md]: I'm also not sure about that `the`. I've seen that Kyma components have it in front of their names (like the Application Connector).

File: docs/github-connector/installation.md (hunk 15 above)
Comment (colunira) [reply docs/github-connector/installation.md]: Chart isn't in guidelines' [list of capitalized terms](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#capitalized-terms). Also, I haven't seen using `the` in front of it.

File: docs/github-connector/installation.md (hunk 14 above)
Comment (colunira) [reply docs/github-connector/installation.md]: Actually, the dot doesn't appear there usually and most people wouldn't put it there, but I wanted to emphasize it. I'll change it to `It consists of the domain name and cannot begin with the dot.`

File: github-connector/README.md (hunk 16 above)
Comment (colunira) [reply github-connector/README.md]: I've changed it to `the Application Reigstry`, ommiting `Kyma` as it is mentioned before and user should know what Application Registry is if he know what Kyma is.

File: docs/github-connector/installation.md (hunk 13 above)
Comment (tomekpapiernik) [inline docs/github-connector/installation.md]:
```suggestion
2. Go to `chart/githubconnector` directory. Run this command to install GitHub Connector:
```
